libs/core/device-select/constants/event.constant.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export enum DeviceSelectEvent {
+  OpenSelectDeviceDrawer = "device-select_open-select-device-drawer",
+  CloseSelectDeviceDrawer = "device-select_close-select-device-drawer",
+}
```

Comment:
⚪ Neutral

What are your thoughts on using this algorithm for Redux actions?
```
DEVICE_SELECT-CLOSE_SELECT_DEVICE_DRAWER
```

DEVICE_SELECT - Event ID
CLOSE_SELECT_DEVICE_DRAWER? - Redux Action Name

Reply:
I am fully convinced, I think I would prefer lowercase :)